index.rst

```diff
@@ -3,11 +3,6 @@
 Symfony Documentation
 =====================
 
-.. toctree::
-    :hidden:
-
-    changelog
```

Comment:
hmm, yeah it would not be generated. I couldn't find any easy way to exclude files for a specific builder.